src/app/workspace/cmdinput/aichat.tsx

```diff
@@ -243,7 +246,8 @@ class AIChat extends React.Component<{}, {}> {
                     this.isFocused.get() ||
                     GlobalModel.inputModel.hasFocus() ||
                     (GlobalModel.getActiveScreen().getFocusType() == "input" &&
-                        GlobalModel.activeMainView.get() == "session")
+                        GlobalModel.activeMainView.get() == "session" &&
+                        GlobalModel.inputModel.getActiveAuxView() == appconst.InputAuxView_AIChat)
```

Comment:
Maybe we should make this logic into a function in inputModel, (`isAuxViewFocused(auxView: InputAuxViewType)`). This could use the logic above and return true or false depending on if the focus is applied.